editor/src/components/editor/component-button.tsx

```diff
@@ -30,19 +30,28 @@ export const ComponentOrInstanceIndicator = React.memo(() => {
     'focusedElementPath',
   )
 
-  const { isComponent, selectedViews } = useEditorState(
+  const autoFocusedPaths = useEditorState(
```

Comment:
It doesn't look like this component is even used anymore, so it can go in the bin